internal/indexer/definitions/torrenthr.yaml

```
@@ -0,0 +1,72 @@
+---
+#id: torrenthr
+name: TorrentHR
+identifier: thr
+description: TorrentHR (THR) is private croatian tracker
+language: hr
+urls:
+  - https://www.torrenthr.org/
+privacy: private
+protocol: torrent
+supports:
+  - irc
+  - rss
+source: unknown
+settings:
+  - name: passkey
+    type: secret
+    required: true
+    label: Passkey
+    help: "Copy the torrent passkey for your account"
+
+irc:
+  network: THR
+  server: irc.torrenthr.org
+  port: 6697
+  tls: true
+  channels:
+    - "#announce"
+  announcers:
+    - THR
+  settings:
+    - name: nick
+      type: text
+      required: true
+      label: Nick
+      help: Bot nick. Eg. user_autodl
+
+    - name: invite_command
+      type: secret
+      default: "THR key IRCKEY"
+      required: true
+      label: Invite command
+      help: Invite auth with THR. Replace IRCKEY.
+
+    - name: auth.account
+      type: text
+      required: false
+      label: NickServ Account. Make sure to group your user and bot.
+      help: NickServ account
```

Mixed up label and help.

```suggestion
      label: NickServ Account. 
      help: NickServ account. Make sure to group your user and bot.
```